.github/workflows/ci.yml

```diff
@@ -2,33 +2,31 @@ name: CI
 
 on:
   push:
-    branches: [ master, ci-* ]
+    branches: [main, ci-*]
   pull_request:
-    branches: [ master ]
+    branches: [main]
 
 jobs:
-
   lint:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
```

Comment:
No changes other than spaces on these - using the defaults of prettierrc in the repo.